portality/models/account.py

```diff
@@ -235,3 +236,11 @@ def new_short_uuid(cls):
             return trunc_uuid
         else:
             return cls.new_short_uuid()
+
+    @classmethod
+    def get_name_safe(cls, account_id) -> str:
```

Comment:
`_safe` is important, it's not design for jinja template, but it have following features:
* high level API
* defensive / robust design 
* always return string
* will not throw exception if account not found 
* easy to use 
* non flexible

`high level API` which mean it easy to use. for example, if developer just want to get name without handling account not found or return None, than you should use `get_name_safe`
`high level API` which also mean that it's not flexible like `low level APi`, for example, developer will never know what is `''` meaning,  because `''` could be account not found or author.name = ''

in case we need a lower level of `get_name` in the future, I can't rename `get_name_safe` in order to avoid confusing.

Reply:
ok, seems like a good argument :)